src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Abstractions/IAirship.cs

```diff
@@ -73,5 +74,9 @@ List<MessageCenter.Message> InboxMessages
         Attributes.AttributeEditor EditChannelAttributes();
 
         Attributes.AttributeEditor EditNamedUserAttributes();
+
+        event EventHandler OnChannelCreation;
```

Comment:
you want these to be `EventHandler<ChannelEventArgs>`. the ChannelEventArgs class will need to be defined outside the interface.